modules/core/src/main/scala/org/scalasteward/core/edit/update/Selector.scala

```diff
@@ -84,6 +88,12 @@ object Selector {
   private def scalaCliUsingLib: Pattern =
     Pattern.compile("""//>\s+using\s+lib\s+""")
 
+  private def scalaCliUsingDep: Pattern =
+    Pattern.compile("""//>\s+using\s+dep\s+""")
+
+  private def scalaCliUsingTestDep: Pattern =
+    Pattern.compile("""//>\s+using\s+test.dep\s+""")
```

Comment:
Not sure about the `.` in `test.dep`. From the [docs](https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html), it looks like the it represents "Any character".

We might have to use escape it.